components/Link.tsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 
 export interface LinkProps {
   href: string;
-  children: string;
+  children: JSX.Element;
```

Comment:
```suggestion
  children: React.ReactNode;
```

Allows for the string too I think